tests/conftest.py

```diff
@@ -54,35 +52,45 @@ def top_dir() -> Path:
 
 
 @pytest.fixture(scope="session")
-def test_data() -> "Dict[str, dict]":
+def test_data() -> "Dict[str, str]":
     """Test data stored in DummyCache."""
+    import json
+
     return {
         # filter
-        "filter-961f5314-9e8e-411e-a216-ba0eb8e8bc6e": {
-            "filterType": "filter/demo",
-            "configuration": {"demo_data": [1, 2]},
-        },
+        "filter-961f5314-9e8e-411e-a216-ba0eb8e8bc6e": json.dumps(
+            {
+                "filterType": "filter/demo",
+                "configuration": {"demo_data": [1, 2]},
+            }
+        ),
         # function
-        "function-a647012a-7ab9-4f2c-9c13-2564aa6d95a1": {
-            "functionType": "function/demo",
-            "configuration": {},
-        },
+        "function-a647012a-7ab9-4f2c-9c13-2564aa6d95a1": json.dumps(
+            {
+                "functionType": "function/demo",
+                "configuration": {},
+            }
+        ),
         # mapping
-        "mapping-a2d6b3d5-9b6b-48a3-8756-ae6d4fd6b81e": {
-            "mappingType": "mapping/demo",
-            "prefixes": {":": "<http://namespace.example.com/ns#"},
-            "triples": [[":a", ":has", ":b"]],
-            "configuration": {},
-        },
+        "mapping-a2d6b3d5-9b6b-48a3-8756-ae6d4fd6b81e": json.dumps(
+            {
+                "mappingType": "mapping/demo",
+                "prefixes": {":": "<http://namespace.example.com/ns#"},
+                "triples": [[":a", ":has", ":b"]],
+                "configuration": {},
+            }
+        ),
         # sessions
-        "1": {"foo": "bar"},
-        "2": {"foo": "bar"},
+        "1": json.dumps({"foo": "bar"}),
```

Comment:
Ah, you moved the `json.dumps()` into here instead. Fair.

Maybe it'd be nicer to store this big dict in a temporary variable then instead, and then return a new dict where we run through that variable passing the values through `json.dumps()`. It makes it slightly easier to add new testdata... Although, I suppose it's the same.
Up to you then :)

Reply:
Ah, I see. You already provided a solution for this. I initially thought about a `pytest.parametrize`-decorator in order to pass the DummyCache-data in the test_**.py itself. But it might also be a bit of overkill and messyin the end ...